bookserver/crud.py

```diff
@@ -157,10 +159,10 @@ async def create_user(user: AuthUserValidator) -> int:
     try:
         async with async_session.begin() as session:
             res = session.add(new_user)
+            return AuthUserValidator.from_orm(res) if res else None
```

Comment:
I've been testing this with the interactive part of the docs tool.  It seems that you need to return from outside of the with statement or else the id field is not filled in on the object. Also res is null so the proper thing to do is to return an AuthUserValidator create from new_user.

Reply:
Good catch -- I didn't think of that (no commit/flush means the transaction hasn't been processed yet.) I didn't understand your second sentence.

Reply:
session.add does not return anything.  But modifies the object but adds the id to the existing object during the commit.